build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+if [ ! -d ./build ]; then
```

Comment:
Just keeping in compatibility with Shadow building, I think having the build directory be an optional argument with a default of `build` would be a good idea.